src/Pkg.jl

```diff
@@ -312,7 +312,9 @@ const build = API.build
     Pkg.pin(pkgs::Union{PackageSpec, Vector{PackageSpec}}; io::IO=stderr, all_pkgs::Bool=false)
 
 Pin a package to the current version (or the one given in the `PackageSpec`) or to a certain
-git revision. A pinned package is never updated. To pin all dependencies set `all_pkgs=true`.
+git revision. A pinned package is never automatically updated: if `pkg` is tracking a path,
+or a repository, those remain tracked in a frozen state). For example, if one wishes to update
+a pinned package in the current manifest, one must unpin, update then pin the package again.
```

Comment:
```suggestion
or a repository, those remain tracked but will not update. 
To get updates from the origin path or remote repository the package must first be freed.
```

Final suggestion.. :)